docs/zq2/index.md

```diff
@@ -4,16 +4,53 @@ title: Zilliqa 2
+
+- **Sharding System with Privacy Features**: Zilliqa 2.0's sharding system supports programmable decentralization, enabling customization of shards for specific use cases. Each shard includes an encryption key, ensuring privacy by encrypting block data.
+
+## Compatibility and Transition
 
-## Networks
+In terms of compatibility and transition:
 
-| Name | EVM Chain Id | API | Explorer | Faucet | Notes |
-| --- | --- | --- | --- | --- | --- |
-| **Zilliqa 2 Devnet** | 33469 | https://api.zq2-devnet.zilstg.dev | https://explorer.zq2-devnet.zilstg.dev | https://faucet.zq2-devnet.zilstg.dev | Active unstable development network. Expect frequent updates, bugs and state resets. |
+- **Existing ZIL and dApps**: Zilliqa 2 ensures a seamless transition by maintaining compatibility with existing Zilliqa and EVM APIs. Users can rest assured that their decentralized applications (dApps) will continue to function normally, with the state of the chain remaining unchanged.
+
+- **Deployment and Stability**: The current phase involves a heavily in-development devnet that aligns with the latest code changes. Users should anticipate some instability and occasional state resets. As Zilliqa 2 matures, the plan is to deploy additional networks offering enhanced stability.
+
+## Deployment and Stability
+
+Presently, there is a heavily in-development devnet that stays aligned with the latest code changes. Users should anticipate some instability and occasional state resets during this phase. As Zilliqa 2 matures, the plan is to deploy additional networks that offer greater stability.
+
+### Networks
+
+| Name                 | EVM Chain Id | API                                 | Explorer                                 | Faucet                                 | Notes                                                                                |
+| -------------------- | ------------ | ----------------------------------- | ---------------------------------------- | -------------------------------------- | ------------------------------------------------------------------------------------ |
+| **Zilliqa 2 Devnet** | 33469        | <https://api.zq2-devnet.zilstg.dev> | <https://explorer.zq2-devnet.zilstg.dev> | <https://faucet.zq2-devnet.zilstg.dev> | Active unstable development network. Expect frequent updates, bugs and state resets. |
+
+## Release Timeline
+
+The release timeline for Zilliqa 2 is contingent on development progress and rigorous testing. Providing an exact date can be challenging due to these variables. To stay abreast of developments and receive timely updates on the release schedule:
+
+- Monitor the [GitHub repository](https://github.com/Zilliqa/zq2)
+- Follow Zilliqa on social media platforms for announcements and progress reports.
+
+For further details and the latest information, stay connected with the Zilliqa community and official channels.
+
+For more information please refer to the [FAQ](faq.md).
```

Comment:
Should we delete the FAQ since this page now covers the same points?

Reply:
Good idea, i will remove it for now. we could bring it back later on if we have more questions to answer